tools/autograd/gen_python_functions.py

```diff
@@ -391,7 +391,7 @@ def emit_namedtuple_typedefs(declarations):
         fn_key = '_'.join(fieldnames)
         fieldsname = flddefnames.get(fn_key)
         if fieldsname is None:
-            fieldsname = 'NamedTuple_fields{}'.format('' if flddefs == [] else len(fielddefs))
+            fieldsname = 'NamedTuple_fields{}'.format('' if flddefs == [] else len(flddefs))
```

Comment:
random unrelated bug I found poking around

Reply:
where's the typechecking 🤣

Reply:
lol, I also noticed and fixed this when I rewrote #46978